In [1]:
x = [ 3.80, 3.80, 3.76, 3.76, 3.72, 3.80, 3.76, 3.76, 3.76, 3.76, 3.76,
3.80, 3.76, 3.76, 3.80, 3.76, 3.80, 3.72, 3.76, 3.80, 3.72, 3.76, 3.80, 3.76,
3.76, 3.76, 3.76, 3.80, 3.76, 3.72 ];
y = [ 3.59, 3.33, 3.32, 3.43, 3.70, 3.77, 3.71, 3.37, 3.77, 3.25, 3.56,
3.78, 3.33, 3.67, 3.76, 3.21, 3.48, 3.74, 3.69, 3.25, 3.44, 3.75, 3.81, 3.31,
3.14, 3.27, 3.33, 3.24, 3.52, 3.43 ];

In [5]:
from scipy import stats
stats.ttest_ind(x,y, equal_var = False)

Ttest_indResult(statistic=6.853428558809812, pvalue=1.3538228149574313e-07)

#### Given a significance of α = 0.01
#### and a p-value of 1.3538 x 10^-7
#### α > p-value and we should reject the null hypothesis.

In [6]:
## Reference: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.ttest_ind.html